var/spack/repos/builtin/packages/composyx/package.py

```diff
@@ -7,16 +7,16 @@
 from spack.package import *
 
 
-class Maphyspp(CMakePackage):
+class Composyx(CMakePackage):
     """a Massively Parallel Hybrid Solver in C++"""
 
-    homepage = "https://gitlab.inria.fr/solverstack/maphys/maphyspp"
-    git = "https://gitlab.inria.fr/solverstack/maphys/maphyspp.git"
-    url = "https://gitlab.inria.fr/api/v4/projects/6194/packages/generic/source/v1.1.9/maphyspp-1.1.9.tar.gz"
+    homepage = "https://gitlab.inria.fr/composyx/composyx"
+    git = "https://gitlab.inria.fr/composyx/composyx.git"
+    url = "https://gitlab.inria.fr/api/v4/projects/52455/packages/generic/source/v1.0.1/composyx-1.0.1.tar.gz"
     maintainers("fpruvost")
 
     version("master", branch="master", submodules=True)
```

Comment:
Looks like the "only" branch at https://gitlab.inria.fr/composyx/composyx at this point for the new repo is `main`.

Reply:
Fixed, thanks !